lib/chef/knife.rb

```diff
@@ -182,7 +182,7 @@ def self.load_config(explicit_config_file, profile)
       config_loader.profile = profile
       config_loader.load
 
-      ui.warn("No knife configuration file found") if config_loader.no_config_found?
+      ui.warn("No knife configuration file found. See https://docs.chef.io/config_rb_knife.html for details.") if config_loader.no_config_found?
```

Comment:
Guiding them to run `knife configure` might be more humane than that doc, which we should also fix.

Reply:
This is just a backport from master. We could probably make it better, but I think this is better than just tossing a message that requires some Google searching